### DISCUSSION TITLE
Stl Surfacer plugin

### DISCUSSION THREAD

**Discussion by LittlePierre** [vlachoudis/bCNC#1781] [Show and tell]
Hello bCNC community 

Is there anybody interested in PR #1561 ? 
Should I close this PR now and keep it for my own use? 
Or commit my latest improvements ?
@Harvie any chance to get it merged ?

It is still experimental, and classified under "development" section of CAM plugins. But it works for most uses, is fast enough, and allows 
- raw slices from a binary stl
- rough milling of a stl, preventing from going into the material,
- surfacing the top face of a stl, preventing from going into the material
- offset from the initial surface

**_Recent screen shots_**
**Raw Slices**
![image](https://user-images.githubusercontent.com/58803300/210029138-19b94609-8e77-4362-a4d6-91ed231533ac.png)

**Rough milling**
![image](https://user-images.githubusercontent.com/58803300/210029184-2ba82a30-bdff-4adc-9acd-ac18a7a60ce2.png)
![image](https://user-images.githubusercontent.com/58803300/210029194-ff54e313-347f-4d09-847d-232955f65a93.png)

**Surfacing**
![image](https://user-images.githubusercontent.com/58803300/210029348-d5f10ef9-f837-4620-8127-15099dd186d4.png)
![image](https://user-images.githubusercontent.com/58803300/210029215-67f59f60-ecac-4427-9ffd-960d3e982ea4.png)

Thanks
Pierre

**Comment by Harvie** [comment]
Yes, i will go through it later. Kinda busy right now, but i am aware of all the pending PRs

**Reply by LittlePierre** [reply]
Thanks @Harvie
